.github/workflows/ci.yml

```diff
@@ -102,6 +102,7 @@ jobs:
       - name: Cache built dependencies
         uses: actions/cache@v3
         with:
+          enableCrossOsArchive: true
```

Comment:
Is this useful to set for the cached build dependencies (and similarly on the CCache files)? AFAICT there shouldn't be anything that can be shared between OSs, because they all compile for different targets. Note that we include `matrix.name` in the cache key, so they won't be used cross-OS anyway (unlike the Sprout parameters for which this is set in the next commit).

Reply:
As you mention, it's not necessary. It was my mistake to copy from the example of the Sprout cache code. The fix will be included in the next commit

Reply:
No need to include a fix; I extracted this change into a separate commit, so you can just rebase and drop that commit.